### PR TITLE
Added initial Doodles support

### PR DESCRIPTION
**Pull request by Ondkloss**
Added initial Doodles support.

### CODE REVIEW

**Review by Rashmi-278 — DISMISSED** [review]

File: protocols/doodles/index.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "doodles",
+  "description": "Doodles is a community-driven collectibles NFT project. Doodle NFTs are made up with over a hundred exciting traits of faces, hair, hats, body and backgrounds. Each Doodle is a unique, non-fungible token (NFT) on the Ethereum blockchain.",
+  "path": "doodles",
+  "isEnabled": true,
+  "type": "snapshot",
+  "safeAddress": "0xDcd382bE6cC4f1971C667ffDa85C7a287605afe4"
```
Comment (sudheerDev) [inline protocols/doodles/index.json:7]: Hey, can you change this to `treasuryAddress: "0xDcd382bE6cC4f1971C667ffDa85C7a287605afe4"`?

Comment (Rashmi-278) [reply protocols/doodles/index.json:7]: @Ondkloss

Comment (Ondkloss) [reply protocols/doodles/index.json:7]: @sudheerDev @Rashmi-278 is `treasuryAddress` a new format? Just to understand. I was working from:
https://docs.boardroom.info/adding-your-project/adding-your-project/protocol-information

Which only mentions `safeAddress`.

Comment (Rashmi-278) [reply protocols/doodles/index.json:7]: @Ondkloss  Yes, we will soon update the docs to reflect that change.

**Review by sudheerDev — APPROVED** [review]

**Comment by Rashmi-278** [comment]
@Ondkloss  Hey, can you please verify the logo you want us to add? the one in resources seems to be doodlebank logo and what we have integrated is doodles nft project.  Do you want us to add this logo https://doodles.app/images/logo-01.svg ?

**Comment by Ondkloss** [comment]
@Rashmi-278 is this for the "little logo circle"? In that case I think the mascot is more recognizable and common. See: https://lh3.googleusercontent.com/7B0qai02OdHA8P_EOVK672qUliyjQdQDGNrACxs7WnTgZAkJa_wWURnIFKeOh5VTf8cfTqW3wQpozGedaC9mteKphEOtztls02RlWQ

Which is used for e.g. for https://twitter.com/doodles and https://opensea.io/collection/doodles-official

**Comment by Rashmi-278** [comment]
Yes exactly ! Thank you for sharing the file will update it

**Pull request merged**